plugins/humanitec-backend/package.json

```diff
@@ -22,8 +27,8 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.21.7",
-    "@backstage/backend-plugin-api": "^0.6.17",
+    "@backstage/backend-common": "^0.22.0",
+    "@backstage/backend-plugin-api": "0.6.18",
```

Comment:
Locking this is required due to https://github.com/janus-idp/backstage-plugins/issues/1843